assets/js/src/poll.jsx

```diff
@@ -0,0 +1,58 @@
+import MailPoet from 'mailpoet';
+
+const getSettingsKey = pollType => `show_poll_success_delivery_${pollType}`;
+
+const initTypeformScript = () => {
+  if (!document.getElementById('typef_orm')) {
+    const js = document.createElement('script');
+    js.id = 'typef_orm';
+    js.src = 'https://embed.typeform.com/embed.js';
+    const q = document.getElementsByTagName('script')[0];
+    q.parentNode.insertBefore(js, q);
+  }
+};
+
+const setPollShown = (pollType) => {
+  const data = {};
+  data[getSettingsKey(pollType)] = '0';
+  MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'settings',
+    action: 'set',
+    data,
+  });
+  // since poll trigger can be called multiple times on single page load
+  // it must be also locally changed in settings
+  window.mailpoet_settings[getSettingsKey(pollType)] = '0';
+};
+
+const Poll = {
+  successDelivery: {
+    canShow: (pollType, skipMtaMethod) => (
+      window.mailpoet_locale === 'en' &&
+      window.mailpoet_settings[getSettingsKey(pollType)] === '1' &&
+      (
+        skipMtaMethod ||
+        (!skipMtaMethod && window.mailpoet_settings.mta.method === 'PHPMail')
```

Comment:
this should be equivalent: `skipMtaMethod || window.mailpoet_settings.mta.method === 'PHPMail'`

Reply:
Fixed